src/utils/spp.spec.ts

```diff
@@ -45,4 +101,94 @@ describe('utils/spp', () => {
     await spp.assertSiteIsContentCenter(siteUrl);
     assert(stubGet.calledOnce);
   });
+
+  it('retrieves model by id', async () => {
+    const stubGet = sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://contoso.sharepoint.com/sites/portal/_api/machinelearning/models/getbyuniqueid('9b1b1e42-794b-4c71-93ac-5ed92488b67f')`) {
+        return model;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await spp.getModel('https://contoso.sharepoint.com/sites/portal', undefined, '9b1b1e42-794b-4c71-93ac-5ed92488b67f');
+    assert(stubGet.calledOnce);
```

Comment:
Instead of doing this, let's check for the output the function generates.

```suggestion
    const actual = await spp.getModel('https://contoso.sharepoint.com/sites/portal', undefined, '9b1b1e42-794b-4c71-93ac-5ed92488b67f');
    assert.deepStrictEqual(actual, expected);
```